app/src/main/AndroidManifest.xml

```diff
@@ -2,13 +2,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.snaplapse">
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
```

Comment:
IDE says I also have to include ACCESS_COARSE_LOCATION to also use fine location, curious if it doesn't apply to you.

Reply:
Yea it says for me too